lib/theming/src/create.ts

```diff
@@ -177,3 +178,35 @@ export const create = (vars: ThemeVar, rest?: Rest): Theme => ({
 
   ...(rest || {}),
 });
+
+export const ensure = (input: any): Theme => {
```

Comment:
I don't like this safety check. I feel like this is going to hide bugs. Can we fail fast here instead?

Reply:
Fail when anything is missing in the theme?

Reply:
Yeah, so the stack trace leads you to the error check, rather than in the depths of some component where it's unclear whether it's a bug in the theme or a bug in the component.

Reply:
I agree on your point that it may hide bugs, but I think throwing on a missing property will cause issues later if we're adding something to the theme.

In that scenario, anything we add to the theme object, will break every theme users have made.

I think some grace is appropriate here, likely the worst that is to happen is, some styling inconsistency in someone's storybook, not broken one.

Reply:
My suggestion:

![screenshot 2019-02-20 at 21 46 41](https://user-images.githubusercontent.com/3070389/53123359-2a53b080-3559-11e9-91f3-3c524605aa9f.png)

The storybook continues to function with the light theme merged in.

Reply:
OK, this looks better to me. Thanks!